CMSgov/mint-app
# [NOREF] Support for removing multiple solutions

# NOREF

## Changes and Description

- Modify `removePlanDocumentSolutionLinks` to take the same parameters as `createPlanDocumentSolutionLinks` (a single solution ID, multiple document IDs)

Follows up on https://github.com/CMSgov/mint-app/pull/296

## How to test this change

- Ensure go tests pass with `scripts/dev up:backend` and `scripts/dev test:go`

- Ensure functionality works as expected
  - Create a model plan
  - Create a custom operational solution
  - Link / unlink multiple documents at once

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
